internal/cmd/intake/runner/list/list.go

```diff
@@ -0,0 +1,179 @@
+
+			// Truncate output
+			if model.Limit != nil && len(runners) > int(*model.Limit) {
+				runners = runners[:*model.Limit]
+			}
+
+			projectLabel := model.ProjectId
+			if len(runners) == 0 {
+				projectLabel, err = projectname.GetProjectName(ctx, p.Printer, p.CliVersion, cmd)
+				if err != nil {
+					p.Printer.Debug(print.ErrorLevel, "get project name: %v", err)
+				}
+			}
+
+			return outputResult(p.Printer, model.OutputFormat, projectLabel, runners)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+// configureFlags adds the --limit flag to the command
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Int64(limitFlag, 0, "Maximum number of entries to list")
+}
+
+// parseInput parses the command flags into a standardized model
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	limit := flags.FlagToInt64Pointer(p, cmd, limitFlag)
+	if limit != nil && *limit < 1 {
+		return nil, &errors.FlagValidationError{
+			Flag:    limitFlag,
+			Details: "must be greater than 0",
+		}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		Limit:           limit,
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+// buildRequest creates the API request to list Intake Runners
+func buildRequest(ctx context.Context, model *inputModel, apiClient *intake.APIClient) intake.ApiListIntakeRunnersRequest {
+	req := apiClient.ListIntakeRunners(ctx, model.ProjectId, model.Region)
+	// Note: we do support API pagination, but for consistency with other services, we fetch all items and apply
+	// client-side limit.
+	// A more advanced implementation could use the --limit flag to set the API's PageSize.
+	return req
+}
+
+// outputResult formats the API response and prints it to the console
+func outputResult(p *print.Printer, outputFormat, projectLabel string, runners []intake.IntakeRunnerResponse) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(runners, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal Intake Runner list: %w", err)
+		}
+		p.Outputln(string(details))
+		return nil
+
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(runners, yaml.IndentSequence(true), yaml.UseJSONMarshaler())
+		if err != nil {
+			return fmt.Errorf("marshal Intake Runner list: %w", err)
+		}
+		p.Outputln(string(details))
+		return nil
+
+	default:
+		if len(runners) == 0 {
+			p.Outputf("No intake runners found for project %q\n", projectLabel)
+			return nil
+		}
+
+		table := tables.NewTable()
+
+		table.SetHeader("ID", "NAME")
+		for i := range runners {
```

Comment:
```suggestion
		for _, runner := range runners {
```

Saves yourself the next line with

```golang
runner := runners[i]
```